tests/test_keys.py

```diff
@@ -164,36 +165,50 @@ def test_import_rsakey_from_public_pem(self):
 
 
   def test_format_metadata_to_key(self):
-    # Reconfiguring rsakey_dict to conform to KEY_SCHEMA
-    # i.e. {keytype: 'rsa', keyval: {public: pub_key, private: priv_key}}
-    keyid = self.rsakey_dict['keyid']
-    del self.rsakey_dict['keyid']
+    # Copying self.rsakey_dict so that rsakey_dict remains
+    # unchanged during and after this test execution.
+    test_rsakey_dict = copy.deepcopy(self.rsakey_dict)
```

Comment:
Nit: I think `copy` would suffice here?

Reply:
Yes, you are right. 

I overdo it a little to make sure the right values were copied but forgot to try with `copy.copy`.

I made the change and amended.